src/reducers.js

```diff
@@ -76,7 +76,8 @@ function organisations(
           .filter(e => {
             return (
               e.roles.find(e => e === "admin") ||
-              e.roles.find(e => e === "supplier")
+              e.roles.find(e => e === "supplier") ||
+              e.roles.find(e => e === "user")
```

Comment:
I think we also need to add "manager" role into the filter as well. The rest is good. I also tested it.